books/free-programming-books-langs.md

```diff
@@ -509,6 +509,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [C++ Programming](https://en.wikibooks.org/wiki/C%2B%2B_Programming) - Panic, et al.
 * [C++ Succinctly, Syncfusion](https://www.syncfusion.com/resources/techportal/ebooks/cplusplus) (PDF, Kindle) (email address *requested*, not required)
 * [C++ Tricks](http://www.bordoon.com/cplusplus/book_wrapper.html)
+* [Competitive Programming Handbook](https://cses.fi/book/book.pdf) - Antti Laaksonen (PDF)
```

Comment:
Plese move it to subjects because it talks about compititive programming concepts more than C++